code/lib/core-common/src/js-package-manager/JsPackageManager.ts

```diff
@@ -8,11 +8,11 @@ import fs from 'fs';
 import dedent from 'ts-dedent';
 import { readFile, writeFile, readFileSync } from 'fs-extra';
 import invariant from 'tiny-invariant';
-import { commandLog } from '../helpers';
+import { commandLog } from '../../../cli/src/helpers';
 import type { PackageJson, PackageJsonWithDepsAndDevDeps } from './PackageJson';
 import storybookPackagesVersions from '../versions';
 import type { InstallationMetadata } from './types';
-import { HandledError } from '../HandledError';
+import { HandledError } from '../../../cli/src/HandledError';
```

Comment:
This references a file inside `@storybook/cli. Is that relative import done on purpose?